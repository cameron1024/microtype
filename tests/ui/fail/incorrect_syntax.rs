fn main() {}

microtype_macro::microtype! {
    String => [Email]
}
