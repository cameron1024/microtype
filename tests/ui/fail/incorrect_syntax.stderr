error: expected curly braces
 --> tests/ui/fail/incorrect_syntax.rs:4:12
  |
4 |     String => [Email]
  |            ^^
